### PR TITLE
added deploy script to make it easier for new devs

### PR DESCRIPTION
**Pull request by talesprates**
This is a script I am developing in order to be able to deploy fifa-champs apps with a single command line. The script usage is available for the user in case he has doubts. The flags the user can set are: MONGOPATH: which is the path where the mongo database is stored; INSTALL: to run npm install in each module before starting the application; and STOP: which will kill all application related processes.

### CODE REVIEW

**Comment by talesprates** [comment]
it also redirects all log messages to FIFACHAMPS/logs/<process>.log, in case the user wants to check it.

File: quickDeploy.sh [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      MONGOPATH=`echo $i | sed -e 's/--mongo=//g'`
+      shift ;;
+    *)
+      throw_arg "'${i}' is not a valid argument."
+      shift ;;
+  esac
+done
+
+if [[ ${STOP} == "true" ]]; then
+  kill $(lsof -t -i:4200) > /dev/null 2>&1
```
Comment (cristiadu) [inline quickDeploy.sh]: This, I know it kills something, probably the backend or frontend, but it isn't very readable and/or understandable.

**Review by cristiadu — CHANGES_REQUESTED** [review]
Deploy is not the same as building it locally. Rename this script.

Also, if possible include a folder with only the .sh files, we have too many scripts right now.

File: quickDeploy.sh (hunk 1 above)
Comment (talesprates) [reply quickDeploy.sh]: well.. i believe i can add comments. but shell script is not very human-friendly to read.

Comment (kiosia) [reply quickDeploy.sh]: I can understand it. It kills whatever process is using port 4200 :p

But then again, I LOVE BASH.

**Comment by kiosia** [comment]
Agreed with Chris comments on namings and directories! For the rest, LGTM!

**Review by kiosia — APPROVED** [review]

File: quickDeploy.sh [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+MONGOPATH="data/db"
+FIFACHAMPS="$(echo $PWD)"
+STOP=false
+
+for i in "$@"
+do
+  case ${i} in
+    -i|--install)
+      INSTALL=true
+      shift ;;
```
Comment (kiosia) [inline quickDeploy.sh]: is this shift really needed here? :thinking: 

I know the one at line 19 is, but I think all the others are not

**Review by kiosia — CHANGES_REQUESTED** [review]

File: quickDeploy.sh (hunk 2 above)
Comment (talesprates) [reply quickDeploy.sh]: makes sense. removed unnecessary shifts.

**Review by kiosia — APPROVED** [review]
lgtm

**Comment by alandoni** [comment]
does it works on Windows? If not, I think it should

**Review by alandoni — CHANGES_REQUESTED** [review]
Make it works on Windows

**Comment by kiosia** [comment]
Not sure, but I think it should work on gitBash.

**Comment by cristiadu** [comment]
Well, fuck windows, am I right? hahahha.

I wouldn't care about it working for general windows, you just need to install bash into your env and everything works, I'm merging this.

**Review by cristiadu — APPROVED** [review]

**Comment by cristiadu** [comment]
Will merge this, a script for Windows will be provided as the issue #141 says.

**Pull request merged**